lnrpc/gen_protos_docker.sh

```diff
@@ -5,8 +5,8 @@ set -e
 # Directory of the script file, independent of where it's called from.
 DIR="$(cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd)"
 
-PROTOBUF_VERSION=$(go list -f '{{.Version}}' -m google.golang.org/protobuf)
-GRPC_GATEWAY_VERSION=$(go list -f '{{.Version}}' -m github.com/grpc-ecosystem/grpc-gateway/v2)
+PROTOBUF_VERSION=$(docker run --rm -v $(pwd)/../:/lnd -w /lnd golang go list -f '{{.Version}}' -m google.golang.org/protobuf)
```

Comment:
Should use `$DIR` instead of `$(pwd)` to make sure this works independently of the directory called from and to avoid yet another sub shell to be invoked. Could also extract the `golang` part into a `GO_IMAGE=golang:1.19.7` env variable to make that a bit more clear and pin the version explicitly to the one we use everywhere else.

Reply:
Yes, unfortunately. But not having a fixed version number has caused issues in the past, so rather have "well defined behavior" instead of things suddenly braking because there's a new Go version out that changes things.

Reply:
Totally agree

Reply:
Done